src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -204,6 +204,10 @@ public class TableMetrics
     public final Counter bytesAnticompacted;
     /** number of bytes where the whole sstable was contained in a repairing range so that we only mutated the repair status */
     public final Counter bytesMutatedAnticompaction;
+    /** number of desynchronized token ranges that were detected during preview repair */
+    public final Counter previewedDesynchronizedTokenRanges;
```

Comment:
A couple of thoughts here:

1. It may also be useful to record the inverse; when we detect that the data is in sync, this would be a good indicator that preview repair is running and finding data in sync.  Guess we would call these `previewedSynchronizedTokenRanges` / `previewedSynchronizedBytes`? 
2. These may also be more useful as Meters instead of Counters, this its more apparent when we stop seeing data out of sync.  e.g. imagine preview repair finds out of sync data, full repair eventually runs, and then we see that data is no longer out of sync.   It may make sense to also apply this to `bytesAnticompacted` and ~bytesMutatedAnticompaction` as well.

Reply:
Looking at the code, I suspect #1 may not be entirely trivial as it looks like the `SyncStatSummary` only includes mismatched bytes in its totals, but it does appear that maybe the data is there in the underlying summaries.

If its not accessible maybe we could benefit from a `previewSuccesses` Counter?